how-to/editor-in-chief-guide.md

```diff
@@ -1,164 +1,297 @@
+:::
 
 ## A note about submissions that are incomplete or vague
 
-In some cases:
+Sometimes a submission lacks enough information to assess whether it’s in
+scope. For example:
 
-* Online documentation of a package is sparse.
-* README is minimal or hard to understand.
-* No clear documentation setup.
-* Elements of the YAML template at the top of the issue are not filled out.
+* Online documentation is sparse
+* The README is minimal or unclear
+* There’s no documentation structure
+* The YAML template at the top of the issue is incomplete
 
-This makes assessment of the package's scope much harder.
-In this case, please ask the author for more information. Even if the package is deemed
-out-of-scope, the package documentation will improve as a result of your questions.
+In these cases, ask the author for more information. Even if the package is
+ultimately out of scope, this will still help improve its documentation.
 
-Example text:
+You can adapt the message below when requesting more details:
 
 ```markdown
-Hello <username> and many thanks for your submission.
+Hello <username>, and thank you for your submission!
 
-We are discussing whether the package is in scope and need a bit more
-information.
+We are reviewing whether your package fits within pyOpenSci’s scope and
+could use a bit more information.
 
-Please add more details and context to your `README` file.
-After reading it, someone with little domain knowledge should understand
-the aim, goals and functionality of the package.
+Please expand your `README` to give more context. After reading it, someone
+without domain expertise should understand the package’s goals, use cases,
+and core functionality.
 
 <optional>
-If a package has overlapping functionality with other packages, we require you
-to mention in your documentation (README) and in this issue [how it is "best in class"](https://www.pyopensci.org/software-peer-review/about/package-scope.html#package-overlap). Please add a more detailed comparison to the packages you mention in the README so we can evaluate?
+If your package overlaps with others, please explain in the README and in
+this issue how your package is "best in class." You can refer to our
+[guidance on package overlap](https://www.pyopensci.org/software-peer-review/about/package-scope.html#package-overlap).
 </optional>
 
 ```
 
 ## Responding to out-of-scope submissions
 
-If the package is determined to be out-of-scope, the Editor in Chief should
-thank authors for their submission, explain the reasons for the decision, and
-direct them to other publication venues if relevant. If further discussion is
-warranted that can take place within the issue.
+If a package is determined to be out of scope, thank the author for their
+submission, briefly explain why it is not in scope, and suggest other venues
+for review if relevant.
+
+Keep the conversation respectful and clear. If the author has questions,
+continue the discussion in the GitHub issue.
+
```

Comment:
Maybe add a section/templates on:

- How to follow up on a stalled review
- How to request clarification from authors